examples/kubernetes/pod.privileged.yaml

```diff
@@ -6,5 +6,23 @@ spec:
   containers:
   - name: buildkitd
     image: moby/buildkit:master
+# the probe below will only work after Release v0.6.3
```

Comment:
works for non-TLS examples

Reply:
also please squash commits, then LGTM

Reply:
So all examples having no certs mounted are the non-TLS?

Reply:
yes

Reply:
adjusted.

Reply:
seems not committed?

Reply:
Sorry, I commited the change.